idaes/power_generation/carbon_capture/mea_solvent_system/unit_models/column.py

```diff
@@ -402,13 +390,17 @@ def _make_performance(self):
 
         self.a_ref = Param(initialize=self.config.packing_specific_area,
                            units=pyunits.m**2 / pyunits.m**3,
-                           doc="Packing specific surface area m2/m3")
+                           doc="Packing specific surface area (m2/m3)")
 
-        self.dh_ref = Expression(expr= 4 * self.eps_ref /self.a_ref,
-                                 doc="Hydraulic diameter")
+        self.dh_ref = Expression(expr=4 * self.eps_ref / self.a_ref,
+                                 doc="Hydraulic diameter (m)")
+        
+        self.S_ref = Param(initialize=self.config.packing_channel_size,
+                           units=pyunits.m,
+                           doc="Packing channel size (m)")
 
-        # specific constants for volumetric mass transfer coefficients
-        # reference:  Billet and Schultes, 1999
+        # Specific constants for volumetric mass transfer coefficients
```

Comment:
I would leave them here for now whilst we get things working, but these should eventually be pulled out in to a sub-model that the user can optionally include.